plugin/phpactor.vim

```diff
@@ -209,7 +209,7 @@ function! phpactor#ClassExpand()
 
     let line = getline('.')
     let char = line[col('.') - 2]
-    let namespace_prefix = classInfo['class_namespace'] . "\\"
+    let namespace_prefix = "\\" . classInfo['class_namespace'] . "\\"
```

Comment:
I'd rather revert this -- I actually don't use this expanding class names in PHP files (I always import) but rather in configuration files (e.g. Symfony DI container configurations). So while this "fixes" it in one respect, it would "break" it in others.